cartridge/roles/vshard-router.lua

```diff
@@ -161,6 +161,7 @@ local function bootstrap_group(group_name, vsgroup)
 
     local conf = confapplier.get_readonly()
     local vshard_cfg = vshard_utils.get_vshard_config(group_name, conf)
+    vshard_cfg.collect_lua_garbage = nil
```

Comment:
Do you try to modify part of read-only config here?

Reply:
`vshard_utils.get_vshard_config` returns a new table, not read only config